routes/reservations.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { Reservation, validate } = require('../models/reservation')
+const router = express.Router()
+
+// TODO: add auth middlewear, when it will be ready
+
+router.get('/', async (req, res) => {
+  const reservations = await Reservation.find()
+
+  res.send(reservations)
+})
+
+router.post('/', async (req, res) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).send(error.details[0].message)
+
+  let reservation = new Reservation({
```

Comment:
i think you can just pass req.body if the body is the same as reservation schema